test/pytest/test_transpose_concat.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+import hls4ml
+import numpy as np
+from tensorflow.keras.models import model_from_json, Model
+from tensorflow.keras.layers import Input, Permute, Concatenate
+import yaml
+
+@pytest.fixture(scope='module')
+def data():
+    X = np.random.rand(1, 2, 3)
```

Comment:
I think we should increase the number of example to test on, bumping up the `1` -> `100` seems sensible, and the test still runs very fast.

Reply:
Done